pkg/gds/config/config.go

```diff
@@ -53,6 +53,11 @@ type AdminConfig struct {
 	TokenKeys map[string]string `split_words:"true"`
 }
 
+type OauthConfig struct {
+	GoogleAudience         string   `split_words:"true"`
+	AuthorizedEmailDomains []string `split_words:"true"`
+}
+
```

Comment:
Moved Oauth configuration away from admin configuration to make it more understandable.